src/harm.cpp

```diff
@@ -78,6 +79,73 @@ void Harm::AddChild(Object *child)
     Modify();
 }
 
+TransPitch Harm::GetRootPitch(unsigned int &pos)
+{
+    Text *textObject = dynamic_cast<Text *>(this->GetChild(0, TEXT));
```

Comment:
Please change method to retrieve the child to `this->FindDescendantByType(TEXT, 1)` for consistency.